sealer/cmd/run.go

```diff
@@ -54,4 +54,6 @@ func init() {
 	runCmd.Flags().StringVarP(&runArgs.Pk, "pk", "", cert.GetUserHomeDir()+"/.ssh/id_rsa", "set baremetal server private key")
 	runCmd.Flags().StringVarP(&runArgs.PkPassword, "pk-passwd", "", "", "set baremetal server  private key password")
 	runCmd.Flags().StringVarP(&runArgs.Interface, "interface", "i", "", "set default network interface name")
+	runCmd.Flags().StringVarP(&runArgs.PodCidr, "podcidr", "", "", "set default pod CIDR network. example '192.168.1.0/24'")
+	runCmd.Flags().StringVarP(&runArgs.SvcCidr, "svccidr", "", "", "Use alternative range of IP address for service VIPs")
```

Comment:
set service IP CIDR. example ...

Reply:
if you dont need a short flag . just use` runCmd.Flags().StringVar()` method.

Reply:
done